python/sbp/client/handler.py

```diff
@@ -167,6 +60,38 @@ def __enter__(self):
   def __exit__(self, *args):
     self.stop()
 
+  # This exception is raised when a message is dispatched to a garbage
+  # collected upstream iterator.
+  class _DeadCallbackException(Exception): pass
```

Comment:
I'm not sure what you're asking.  It's an internal exception that is an implementation detail to allow dead iterators to be garbage collected.  It's not intended for public use.